metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -93,7 +93,7 @@ public KRaftMigrationDriver(
         this.log = LoggerFactory.getLogger(KRaftMigrationDriver.class);
         this.migrationState = MigrationDriverState.UNINITIALIZED;
         this.migrationLeadershipState = ZkMigrationLeadershipState.EMPTY;
-        this.eventQueue = new KafkaEventQueue(Time.SYSTEM, new LogContext("KRaftMigrationDriver"), "kraft-migration");
+        this.eventQueue = new KafkaEventQueue(Time.SYSTEM, new LogContext("KRaftMigrationDriver "), "kraft-migration");
```

Comment:
can we add nodeId here to the LogContext? Maybe like "[KRaftMigrationDriver nodeId=1] " etc. etc. to match our usual style in the controller.

Reply:
btw `KRaftMigrationDriver.log` should also be created by this logContext, so we know what node our logs are coming from.